config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/ingredients", to: "ingredients#index"
+
+  get "/recipes/:id", to: "recipes#show"
+  post "/recipes/:id", to: "recipes#create_ingredient"
```

Comment:
The path `/recipes/:id` is not the most RESTful option for this create action. We are creating an association between two resources, so it's a good idea to include both the resource names in the path. A better option could be `/recipe_ingredients` or `/recipes/:id/ingredients` so that you can pass the recipe ID. Both are RESTful

Also, with the controller suggestion I mentioned above, you'd want to direct this route to a different controller. Take together, this route could instead be:

  post "/recipes/:id/ingredients", to: "recipe_ingredients#create"